### PR TITLE
chore: rename lib to dockest; rename root to dockest-monorepo

### PR DESCRIPTION
**Pull request by n1ru4l**
I think this makes more sense, as the lib package (which should actually be the dockest package) should be published.

### CODE REVIEW

**Comment by erikengervall** [comment]
> I think this makes more sense, as the lib package (which should actually be the dockest package) should be published.

Yep, I agree 👍

File: packages/dockest/src/onRun/waitUntilContainersAreReady.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      service.logger.info('Received start event.')
+      resolve()
+    }
+  }
+
+  emitter.addListener(service.runnerConfig.service, listener)
+  service.logger.info('Waiting for start event.')
+
+  return new Promise(_resolve => (resolve = _resolve))
+}
```
Comment (erikengervall) [inline packages/dockest/src/onRun/waitUntilContainersAreReady.ts]: Could you introduce this file in a separate PR?

**Comment by erikengervall** [comment]
> @erikengervall I think you have to update the snyk config 😅

Haha yeah, it looks pretty broken 😅

File: packages/dockest/src/onRun/waitUntilContainersAreReady.ts (hunk 1 above)
Comment (n1ru4l) [reply packages/dockest/src/onRun/waitUntilContainersAreReady.ts]: oooops

Comment (n1ru4l) [reply packages/dockest/src/onRun/waitUntilContainersAreReady.ts]: yeah sure, no idea how this ended up in this pr 😅

Comment (n1ru4l) [reply packages/dockest/src/onRun/waitUntilContainersAreReady.ts]: rebased and removed ✅

**Comment by erikengervall** [comment]
[`@examples/compose-files-only: dockest.ts(1,35): error TS2307: Cannot find module '../../lib/src'.`](https://travis-ci.com/erikengervall/dockest/builds/131085721#L583)

Looks like some paths needs to be updated due to `lib` -> `dockest` refactor. Would one be able to import dockest since it's included as a devDep in each example?

**Comment by n1ru4l** [comment]
@erikengervall yarn workspaces should automatically link dockest to the packages. Therefore we should be able to to import from "dockest" directly.

File: packages/examples/compose-files-only/package.json [hunk 2]
```diff
@@ -15,6 +15,8 @@
   "devDependencies": {
     "@types/ioredis": "^4.0.13",
     "@types/pg": "^7.11.1",
-    "@types/sequelize": "^4.28.4"
+    "@types/sequelize": "^4.28.4",
+    "dockest": "1.0.0",
```
Comment (erikengervall) [inline packages/examples/compose-files-only/package.json]:
```suggestion
    "dockest": "*",
```
To align with the other examples

File: package.json [hunk 3]
```diff
@@ -1,38 +1,25 @@
...
     "type": "git",
     "url": "https://github.com/erikengervall/dockest.git"
   },
   "scripts": {
     "clean": "lerna clean --yes && rimraf ./dist ./node_modules && yarn",
     "build": "lerna run build --stream",
-    "lint": "eslint \"./packages/lib/**/*.ts\"",
+    "lint": "eslint \"./packages/**/*.ts\"",
     "test": "lerna run test --stream",
     "generate:typedoc": "typedoc ./packages/lib"
```
Comment (erikengervall) [inline package.json]:
```suggestion
    "generate:typedoc": "typedoc ./packages/dockest"
```

Comment (erikengervall) [reply package.json]: This'll fix the typedoc generation in the CI

**Comment by erikengervall** [comment]
I added Snyk as a devDep in the project so that one can run in locally.

Also updated the manifest list at Snyk.io to point towards the correct package.json. AFAIK there's no automatic process on Snyk's end to detect updates in the manifest list and it'll just fail if a package.json is updated/removed.

**Comment by n1ru4l** [comment]
Finally 🎉

thank u Travis

**Comment by erikengervall** [comment]
> Finally 🎉
> 
> thank u Travis

Introduced some conflicts with your PR: sorry. 

Also, doesn't seem to be building with GitHub Actions 😕 

Other than that - I'm happy with the changes 👍

**Comment by n1ru4l** [comment]
@erikengervall JObs did run on my own repo, dunno why they did not run here...

**Comment by erikengervall** [comment]
> > @erikengervall Jobs did run on my own repo (https://github.com/n1ru4l/dockest/runs/258774218), dunno why they did not run here...
> > I rebase upon master!
> 
> Weird that nothing but `Travis CI - Pull Request` turns up on this page 😕
> 
> Hard to enforce `Require status checks to pass before merging` if this persists, since I cannot merge until all CI tests runs successfully.

I'll temporarily disable requirements for status checks for everything _but_ Travis PR, since that seem to work 😕

**Pull request merged**